test/cli-build.spec.js

```diff
@@ -135,6 +137,7 @@ describe('cli build', () => {
           expect(removeSpy).toHaveBeenCalledWith(
             skyPagesConfigUtil.spaPathTemp()
           );
+          expect(passedConfig.skyuxPathAlias).toBeUndefined();
```

Comment:
Does `toBeUndefined` pass if the property is there but has the value of `undefined`?  Maybe this should be `expect(passedConfig.hasOwnProperty('skyuxPathAlias')).toBe(false);` instead since the original bug was that the property was present but undefined.